<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1078</width>
    <height>514</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res.qrc">
    <normaloff>:/Image/院徽.png</normaloff>:/Image/院徽.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <widget class="QWidget" name="widget_8" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QTableWidget" name="tableWidget">
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <column>
          <property name="text">
           <string>ID</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>DLC</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>HEX</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>名称</string>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget_3" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QWidget" name="widget_2" native="true">
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <widget class="QLabel" name="label">
               <property name="text">
                <string>主动发送</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_7">
               <property name="text">
                <string>ID</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit_2"/>
             </item>
             <item>
              <widget class="QLabel" name="label_8">
               <property name="text">
                <string>DLC</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit_3"/>
             </item>
             <item>
              <widget class="QLabel" name="label_9">
               <property name="text">
                <string>HEX</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit"/>
             </item>
             <item>
              <widget class="QPushButton" name="writeButton">
               <property name="text">
                <string>Send</string>
               </property>
               <property name="icon">
                <iconset resource="res.qrc">
                 <normaloff>:/Image/send.png</normaloff>:/Image/send.png</iconset>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="widget" native="true">
            <layout class="QVBoxLayout" name="verticalLayout">
             <item>
              <widget class="QPushButton" name="findPortButton">
               <property name="text">
                <string>搜索串口</string>
               </property>
               <property name="icon">
                <iconset resource="res.qrc">
                 <normaloff>:/Image/search.png</normaloff>:/Image/search.png</iconset>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_5">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>端口</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="serialPortComboBox"/>
             </item>
             <item>
              <widget class="QLabel" name="label_6">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>波特率</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="baudComboBox"/>
             </item>
             <item>
              <widget class="QPushButton" name="connectPushButton">
               <property name="text">
                <string>连接</string>
               </property>
               <property name="icon">
                <iconset resource="res.qrc">
                 <normaloff>:/Image/connect.png</normaloff>:/Image/connect.png</iconset>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="show">
               <property name="text">
                <string>显示输入输出</string>
               </property>
               <property name="icon">
                <iconset resource="res.qrc">
                 <normaloff>:/Image/status.png</normaloff>:/Image/status.png</iconset>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget_7" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>设备状态:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>\</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>设备ID:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>\</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_22">
         <property name="text">
          <string>启动从站</string>
         </property>
         <property name="icon">
          <iconset resource="res.qrc">
           <normaloff>:/Image/act.png</normaloff>:/Image/act.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_21">
         <property name="text">
          <string>debug</string>
         </property>
         <property name="icon">
          <iconset resource="res.qrc">
           <normaloff>:/Image/set.png</normaloff>:/Image/set.png</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget_4" native="true">
      <layout class="QVBoxLayout" name="verticalLayout_3"/>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
